src/app/main/header/header.component.ts

```diff
@@ -61,4 +64,14 @@ export class HeaderComponent {
         this.show = !this.show;
     }
 
+    doSearch(search: SearchParams) {
+        // reset search params
+        this.searchParams = undefined;
+        // we can do the routing here or send the search param
+        // to (resource) list view directly
+        this.searchParams = search;
+
+        this._router.navigate(['/search/' + this.searchParams.mode + '/' + encodeURIComponent(this.searchParams.query)]);
```

Comment:
In case of selected project we have to add the project iri to the search route. For example:

```typescript
let doSearchRoute = '/search/' + this.searchParams.mode + '/' + encodeURIComponent(this.searchParams.query);
if(this.searchParams.filter.limitToProject) {
  doSearchRoute += '/' + encodeURIComponent(this.searchParams.filter.limitToProject);
}

this._router.navigate([doSearchRoute]);

```

Reply:
Done